chromium_src/chrome/installer/setup/uninstall.cc

```diff
@@ -49,17 +52,25 @@ InstallStatus UninstallProduct(const ModifyParams& modify_params,
                                const base::CommandLine& cmd_line) {
   DeleteBraveFileKeys(HKEY_CURRENT_USER);
 
-  const InstallerState& installer_state = *modify_params.installer_state;
+  const auto installer_state = modify_params.installer_state;
   const base::FilePath chrome_exe(
-      installer_state.target_path().Append(installer::kChromeExe));
+      installer_state->target_path().Append(installer::kChromeExe));
   const std::wstring suffix(
       ShellUtil::GetCurrentInstallationSuffix(chrome_exe));
-  if (installer_state.system_install() ||
+  if (installer_state->system_install() ||
       (remove_all &&
        ShellUtil::QuickIsChromeRegisteredInHKLM(chrome_exe, suffix))) {
     DeleteBraveFileKeys(HKEY_LOCAL_MACHINE);
   }
-
+#if BUILDFLAG(ENABLE_BRAVE_VPN_SERVICE)
```

Comment:
I think uninstall code should be available always.
If not, some users will have service forever even after unintall with below situation.
1. Installed and registered dns filter service.
2. Brave is updated w/o this uninstall code
3. User uninstalled and filter service is remained.

Reply:
ok